kura/examples/org.eclipse.kura.example.wire.math.singleport.provider/OSGI-INF/metatype/org.eclipse.kura.wire.GainOffset.xml

```diff
@@ -16,7 +16,7 @@
             name="configuration"
             type="String" cardinality="0"
             required="true"
-            default="toBeMultipliedByTwo | 2&#13;toBeMultipliedBy3AndIncreasedBy1 | 3 | 1"
+            default="toBeMultipliedByTwo | 2&#10;toBeMultipliedBy3AndIncreasedBy1 | 3 | 1"
```

Comment:
Without this change, tests can either be ineffective/useless or will not work for this component.
To me it does not make sense to either:

- push tests only for the new components
- push tests also for the old components, but which fail because of the way tests work
- push tests also for the old components, which do not actually perform useful tests